cmd/api-linter/cli.go

```diff
@@ -180,7 +185,8 @@ func (c *cli) lint(rules lint.RuleRegistry, configs lint.Configs) error {
 	}
 
 	// Create a linter to lint the file descriptors.
-	l := lint.New(rules, configs, lint.Debug(c.DebugFlag))
+	l := lint.New(rules, configs, lint.Debug(c.DebugFlag),
+		lint.IgnoreCommentDisables(c.IgnoreCommentDisablesFlag))
```

Comment:
Go does not have a line length restriction. I would suggest putting this on one line. It would be more standard.

Reply:
sure, no opinion on line length.